tests/cpp.test

```diff
@@ -970,3 +970,4 @@
 60054  interface-keyword-in-cpp.cfg         cpp/interface-keyword-in-cpp.cpp
 60055  issue_3116.cfg                       cpp/issue_3116.cpp
 60056  issue_3116.cfg                       cpp/issue_3116-2.cpp
+60057  issue_3116-2.cfg                     cpp/issue_3116-3.cpp
```

Comment:
Is there any reason `cpp/issue_3116.cpp` can't be recycled for this purpose?

Reply:
I suppose my main concern was that `cpp/issue_3116.cpp` also contains cases which weren't mentioned with the config used for this issue, so I don't know what the "expected" format would be. But if you don't deem that to be an issue then yes, we can definitely re-use it.

Reply:
That feels like a reason *to* combine them, so we can look at the output with additional cases and verify it looks reasonable 🙂.

Reply:
Good point, updated the PR with the test replaced by `cpp/issue_3116.cpp`.